packages/eslint-plugin/docs/rules/consistent-type-exports.md

```diff
@@ -70,15 +101,15 @@ export type { ButtonProps } from 'some-library';
 ### ✅ Correct
 
 ```ts
-export { Button } from 'some-library';
-export type { ButtonProps } from 'some-library';
 export { Button, type ButtonProps } from 'some-library';
 ```
 
 ## When Not To Use It
 
 - If you are using a TypeScript version less than 3.8, then you will not be able to use this rule as type exports are not supported.
 - If you specifically want to use both export kinds for stylistic reasons, you can disable this rule.
+- If you use `--isolatedModules` the compiler would error if a type is not re-exported using `export type`.
+- If you also don't wish to enforce one style over the other, you can disable this rule.
```

Comment:
@Josh-Cena my intent here was for these to be on the same line. 

```suggestion
- If you use `--isolatedModules` the compiler would error if a type is not re-exported using `export type`. If you also don't wish to enforce one style over the other, you can disable this rule.
```

Reply:
Really...? They look like completely disjoint points though.